ci-operator/jobs/red-hat-data-services/opendatahub-operator/red-hat-data-services-opendatahub-operator-v1.0.0-periodics.yaml

```diff
@@ -0,0 +1,115 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 30 1 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: v1.0.0
+    org: red-hat-data-services
+    repo: opendatahub-operator
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-red-hat-data-services-opendatahub-operator-v1.0.0-modh-opendatahub-operator-nightly-v100
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --report-credentials-file=/etc/report/credentials
+      - --target=modh-opendatahub-operator-nightly-v100
+      command:
+      - ci-operator
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /secrets/gcs
+        name: gcs-credentials
+        readOnly: true
+      - mountPath: /etc/pull-secret
+        name: pull-secret
+        readOnly: true
+      - mountPath: /etc/report
+        name: result-aggregator
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build01
+  cron: 30 3 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: v1.0.0
```

Comment:
should it be 1.0.1 now?

Reply:
The branch is still the v1.0.0 branch.  Each release in the series is a new tag at the end of the branch.